entry/src/main/js/default/pages/index/index.js

```diff
@@ -0,0 +1,23 @@
+export default {
+    data: {
+        weather: "sun & windy",
+        // cloudy, windy, partly sunny, rainy, sleeting, sun & rain, sun & windy, sunny, thunderstorm & rain, thunderstorm
```

Comment:
UI functionality is fine, please include the api exposing functionality where in data layer need to be segregated as per the requirement.